client/public/index.html

```diff
@@ -9,6 +9,8 @@
   />
   <meta name="theme-color" content="#000000" />
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
```

Comment:
Hey, Could we take this out? I think we are using another font.

Reply:
The font is taken out! Should be good now.